agenthub/thinkact_agent/plans.md

```diff
@@ -0,0 +1,36 @@
+# Things that need to be done for SWE-Agent
+
+## 1. Add in all of the following commands to ACI
+```
+'search_for': {
+    'params': '<keywords>',
+    'description': 'Will allow you to search your working directory for files and folders that match your <keyword>.',
+}
```

Comment:
Definitely want to add a `search_for` command. IMO it should be called `search_code` so the LLM doesn't confuse it with searching on Google